src/rand.rs

```diff
@@ -229,6 +232,23 @@ mod sysrand_chunk {
     }
 }
 
+#[cfg(target_os = "openbsd")]
+mod sysrand_chunk {
+    use core::cmp::min;
+    use crate::{c, error};
+
+    #[inline]
+    pub fn chunk(dest: &mut [u8]) -> Result<usize, error::Unspecified> {
+        let chunk_len: c::size_t = min(dest.len(), 256);
+        // See `SYS_getentropy` in #include <sys/syscall.h>.
+        let r = unsafe { libc::syscall(7, dest.as_mut_ptr(), chunk_len) };
```

Comment:
Instead of manually using `syscall`, why not just use [`libc::getentropy`](https://rust-lang.github.io/libc/x86_64-unknown-openbsd/doc/libc/fn.getentropy.html)

Reply:
Fixed up to use libc::getentropy instead.